README.md

```diff
@@ -1,2 +1,288 @@
+
+    L3 --> Length from the Hip joint to the Mechanism's axis of Symmetry.
+    </font>
+
+* **Equations Used To Derive the System Design Variables**
+
+    ![](https://i.imgur.com/E1Gui6i.png)
+
+    In Above Image we have equations to calculate the <font color="yellow">$\phi$1, $\phi$2, L1, L2 and L3 </font>.
+
+    ![](https://i.imgur.com/1IjtgOg.png)
+
+    In above Image, it is the General Equation of Length (vertical distance between the axis of the hip joints and the foot joint)
+    Here, G (Gear Ratio) which is taken to be 1 for our Bot.
+    
+    
+    ![](https://i.imgur.com/9046wKp.png)
+
+    The above Graph gives is about change of Leg Extension wrt change in Driver Motor Angle. 
+
+
+    [Link To Python Script Used]()
+    [LInk to Python Script to Plot the Graph]()
+
+* **Specfications of Bot**
+    <font color="red">
+    
+    Lmax = 13 cm
+
+    Lmin = 4 cm
+    
+    L1 = 4.286 cm
+    
+    L2 = 9.236 cm
+
+    L3 = 3.5 cm
+
+    Phi1 = 45
+
+    Phi2 = 15
+
+    </font>
+
+
+
+
+## <ins> Components
+* Bot Parts of Acrylic Material (Laser-Cutted)
+* N20 motor Shaft for Wheels (3D Printed)
+* SRA PCB 
+* N20 motors
+* MG90 Servo Motors
+* Buck Converter (LM2596)
+* Bearings
+* Shafts
+
+> :memo: **Note:** [Link to Components used In Robot and Description](https://github.com/KudoC0nan/Balance_On_Wheels/blob/dev_arsh/Notes/Components.md)
+
+## <ins> PID
+A PID controller is one kind of device used to control different process variables like pressure, flow, temperature, and speed in industrial applications. 
+
+> :memo: **Note:** [PID Notes](https://github.com/KudoC0nan/Balance_On_Wheels/blob/dev_arsh/Notes/PID%20Explanation)
+
+## <ins> Code Flow
+
+Here, we First wrote the algorithm for Self-Balancing the bot.
+
+
+For self-balancing the bot, we need to keep the bot a bit above our desired angle (Setpoint), since the bot is very fast and will fall down once it's past the desired angle.
+
+The pitch angle is required for Self-Balancing which is obtained from the Inertial Measurement Unit(IMU sensor) plug-in.
+
+> :memo: **Note:**[Link to Explanation of Self Balancing Code](https://github.com/KudoC0nan/Balance_On_Wheels/blob/dev_arsh/Notes/Self%20Balancing%20Code.md)
+
+
+And after Successful Run of Self Balancing Code, we than
+wrote the code for Servo moving from Maximum to Minimum Length (Variable Height).
+
+[Link to Code]()
+
+And after that we wrote the code of integrating Self Balancing Code with servos locking up at minimum height for Self Balancing of Bot at Minimum Height.
+
+[Link to Code](https://github.com/KudoC0nan/Balance_On_Wheels/tree/dev_arsh/Self_Balancing_Servo_set_zero)
+
+**Flow Chart of Code**
+![](https://i.imgur.com/6f7omtf.png)
+
+After Which we wrote the Jumping Code for Bot to Jump from Minimum Position.
+
+[Link to Code]()
+
+After Which we wrote the code for integration of Jumping Code with Self Balancing Code.
+
+[Link to Code]()
+
+---
+## <ins>Results and Demo
```

Comment:
Add videos for each functionality

Reply:
Added videos for each functionality except jumping and self balance simultaneously